docs/source/changes.rst

```diff
@@ -5,6 +5,47 @@
 Changes
 =======
 
+v0.5.0
+======
+
+**Breaking change**
+
+* Remove the ``BoxCox`` transform. ( `#222`_ )
+
+**New features**
+
+* Add new contribution ``iSMD`` by `qi`_. ( `#208`_ )
+* Add classification support for ``Trainer``. ( `#184`_ )
+
+**Enhance**
+
+* ``BaseFeaturizer.transform`` now supports ``pandas.DataFrame`` as input. (??)
```

Comment:
* ``BaseFeaturizer.transform`` now supports ``pandas.DataFrame`` as input where relevant columns for descriptor calculation can be specified through ``target_col``.
* ``BaseDescriptor.transform`` and ``BaseLogLikelihoodSet.log_likelihood`` now automatically check if any group names occur in the input ``pandas.DataFrame`` column names. If not, the entire ``pandas.DataFrame`` will be passed to the corresponding ``BaseFeaturizer`` and ``BaseLogLikelihood``, respectively.

Reply:
comments have been added